dali/kernels/signal/fft/cufft_helper.cc

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/kernels/signal/fft/cufft_helper.h"
+
+namespace dali {
+
+DLL_PUBLIC int cufftGetVersion() {
+  int major = -1;
+  int minor = -1;
+  int patch = -1;
+  auto ret = cufftGetProperty(MAJOR_VERSION, &major);
+  if (ret != CUFFT_SUCCESS) {
+    return -1;
+  }
+  ret = cufftGetProperty(MAJOR_VERSION, &minor);
+  if (ret != CUFFT_SUCCESS) {
+    return -1;
+  }
+  ret = cufftGetProperty(PATCH_LEVEL, &patch);
+  if (ret != CUFFT_SUCCESS) {
+    return -1;
+  }
+  return major*1000 + minor*10 + patch;
```

Comment:
I've seen this bit of magic in few places in this PR. Maybe it would be good to extract in and call `GetVersionNumber` or whatever similar, so it is clear, why this happens like this?

For me it's ok how it is now, but I know how these versions are numbered.

Reply:
Done for cuFFT and nvJPEG. NPP follows different conventions to obtain major, minor, and patch values, still the function that combines them to one value is now common.